### PR TITLE
Jenkins 7 - Edit README

### PR DESCRIPTION
**Pull request by caleonardo**
This is to address Issue https://github.com/terraform-google-modules/terraform-example-foundation/issues/192

Thank you @daniel-cit

### CODE REVIEW

File: 1-org/README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ You can choose not to enable the Data Access logs by setting variable `data_acce
     _TF_SA_EMAIL
     _STATE_BUCKET_NAME
     ```
-1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . `1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
+1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
```
Comment (atos-cit) [inline 1-org/README.md:43]: The phrase `to the root of your new repository (modify accordingly based on your current directory).` is repeated twice.

File: 1-org/README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ You can choose not to enable the Data Access logs by setting variable `data_acce
 1. Navigate into the repo `cd gcp-org` and change to a non production branch `git checkout -b plan`
 1. Copy contents of foundation to new repo `cp -RT ../terraform-example-foundation/1-org/ .` (modify accordingly based on your current directory).
 1. Copy cloud build configuration files for terraform `cp ../terraform-example-foundation/build/cloudbuild-tf-* . ` (modify accordingly based on your current directory).
-1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . `1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
+1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
```
Comment (atos-cit) [inline 1-org/README.md:23]: The phrase `to the root of your new repository (modify accordingly based on your current directory).` is repeated twice.

File: 2-environments/README.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ The purpose of this step is to set updevelopment,non-production, and production
 1. Change freshly cloned repo and change to non master branch `git checkout -b plan`
 1. Copy contents of foundation to new repo `cp -RT ../terraform-example-foundation/2-environments/ .` (modify accordingly based on your current directory)
 1. Copy cloud build configuration files for terraform `cp ../terraform-example-foundation/build/cloudbuild-tf-* . ` (modify accordingly based on your current directory).
-1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . `1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
+1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
```
Comment (atos-cit) [inline 2-environments/README.md:19]: The phrase `to the root of your new repository (modify accordingly based on your current directory).` is repeated twice.

File: 2-environments/README.md [hunk 4]
```diff
@@ -42,7 +42,7 @@ The purpose of this step is to set updevelopment,non-production, and production
     _TF_SA_EMAIL
     _STATE_BUCKET_NAME
     ```
-1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . `1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
+1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
```
Comment (atos-cit) [inline 2-environments/README.md:45]: The phrase `to the root of your new repository (modify accordingly based on your current directory).` is repeated twice.

File: 3-networks/README.md [hunk 5]
```diff
@@ -31,7 +31,7 @@ If you are not able to use dedicated interconnect, you can also use an HA VPN to
 1. Change freshly cloned repo and change to non master branch `git checkout -b plan`
 1. Copy contents of foundation to new repo `cp -RT ../terraform-example-foundation/3-networks/ .` (modify accordingly based on your current directory).
 1. Copy cloud build configuration files for terraform `cp ../terraform-example-foundation/build/cloudbuild-tf-* . ` (modify accordingly based on your current directory)
-1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . `1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
+1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
```
Comment (atos-cit) [inline 3-networks/README.md:34]: The phrase `to the root of your new repository (modify accordingly based on your current directory).` is repeated twice.

File: 3-networks/README.md [hunk 6]
```diff
@@ -65,7 +65,7 @@ If you are not able to use dedicated interconnect, you can also use an HA VPN to
     _TF_SA_EMAIL
     _STATE_BUCKET_NAME
     ```
-1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . `1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
+1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
```
Comment (atos-cit) [inline 3-networks/README.md:68]: The phrase `to the root of your new repository (modify accordingly based on your current directory).` is repeated twice.

File: 4-projects/README.md [hunk 7]
```diff
@@ -48,7 +48,7 @@ If your user does not have access to run the commands above and you are in the o
     _TF_SA_EMAIL
     _STATE_BUCKET_NAME
     ```
-1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . `1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
+1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
```
Comment (atos-cit) [inline 4-projects/README.md:51]: The phrase `to the root of your new repository (modify accordingly based on your current directory).` is repeated twice.

File: 1-org/README.md (hunk 1 above)
Comment (caleonardo) [reply 1-org/README.md:43]: Will open another PR for this issue. Thank you!

**Comment by caleonardo** [comment]
The PR https://github.com/terraform-google-modules/terraform-example-foundation/pull/201 replaces this PR https://github.com/terraform-google-modules/terraform-example-foundation/pull/197

**Pull request closed without merging**